examples/apps/tree-comparison/src/model/newTreeInventoryList.ts

```diff
@@ -127,29 +174,21 @@ export class NewTreeInventoryList extends DataObject implements IInventoryList {
 			allowedSchemaModifications: AllowedUpdateType.None,
 			schema,
 		});
-		// REV: This event feels overly-broad for what I'm looking for, but I'm having issues with
-		// more node-specific events ("changing", etc.).  I also personally find the deviation from
-		// standard EventEmitter API surprising/unintuitive/inconvenient.
+		// REV: This event feels overly-broad for what I'm looking for - is there a more localized event that
```

Comment:
This comment is a little stale since I was able to use the "changing" event for quantity changes, but it's still tough to work with.  See in particular the weird "filtering" logic I have to do below to figure out what items got deleted from the list.

Ideally I think I'd like to register for an event on the inventoryItems (list) for when that specific node (not subtree) has items added to or removed from the list, and ideally with a param indicating the added/removed items.  I don't yet have a strong opinion on whether that would be better as a single "listItemsChanged" type event, or more-specific "listItemsInserted"/"listItemsRemoved" - the latter might be easier to consume here at least?

Reply:
e.g. here I just want to handle items being added/deleted, but the handler also fires whenever item quantities update and I need to think about and be careful to not respond in those cases.

Reply:
Interesting - that is great feedback. We currently don't have events beyond `afterChange` and `beforeChange` on our radar (`changing` is going away), so this will be a helpful usecase in evaluating what else we ought to add. Are you ignoring the rest of the subtree below the list elements for performance, or correctness, or both?

Reply:
Purely correctness, I'm not concerned with performance since this is a 101 demo.

FWIW I think I almost never want events for the subtree, and instead just want them for the target node.  Handling events for the subtree requires me to know EVERY possible type of change happening in the subtree to ensure I'm responding in the correct way for each one